src/components/Practice/PhrasesInAction.jsx

```
@@ -5997,368 +5991,268 @@ const PhrasesInAction = ({
+        alt="hint"
+        style={{
+          width: "50px",
+          height: "50px",
+          position: "absolute",
+          top: "20px",
+          left: "20px",
+          cursor: "pointer",
+          zIndex: 1000,
+        }}
+        onClick={() => setOpen(true)}
+      />
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)